src/nvim/path.c

```diff
@@ -1749,12 +1749,28 @@ char_u *find_file_name_in_path(char_u *ptr, size_t len, int options, long count,
   return file_name;
 }
 
-// Check if the "://" of a URL is at the pointer, return URL_SLASH.
+/// Checks for a Windows drive letter ("C:/") at the start of the path.
+///
+/// @see https://url.spec.whatwg.org/#start-with-a-windows-drive-letter
+bool path_has_drive_letter(const char *p)
+  FUNC_ATTR_NONNULL_ALL
+{
+  // https://url.spec.whatwg.org/#start-with-a-windows-drive-letter
+    return strlen(p)>=2
```

Comment:
- this indentation will fail the lint CI. Please check what lint CI reports
- don't need the comment now since it's in the docstring